src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/experimental_detectron_detection_output_ref.cl

```diff
@@ -138,21 +138,28 @@ KERNEL(eddo_ref_stage_0)
     size_t class_idx = get_global_id(1);
 #endif
 
-    INPUT_TYPE4 box = vload4(roi_idx, boxes);
+    INPUT_TYPE4 box;
+    box[0] = boxes[INPUT0_GET_INDEX(roi_idx, 0, 0, 0)];
+    box[1] = boxes[INPUT0_GET_INDEX(roi_idx, 1, 0, 0)];
+    box[2] = boxes[INPUT0_GET_INDEX(roi_idx, 2, 0, 0)];
+    box[3] = boxes[INPUT0_GET_INDEX(roi_idx, 3, 0, 0)];
 
     if (any(islessequal(box.hi - box.lo, ZERO2))) {
         const int refined_offset = roi_count * class_idx + roi_idx;
         refined_scores[refined_offset] = ZERO;
     } else {
-        const int offset = NUM_CLASSES * roi_idx + class_idx;
-
         // width & height of box
         INPUT_TYPE2 box_size = (box.hi - box.lo + COORDINATE_OFFSET);
 
         // center location of box
         const INPUT_TYPE2 center = box.lo + HALF_ONE * box_size;
 
-        const INPUT_TYPE4 delta = vload4(offset, deltas) / DELTA_WEIGHTS;
+        INPUT_TYPE4 delta;
+        delta[0] = deltas[INPUT1_GET_INDEX(roi_idx, class_idx * 4, 0, 0)];
```

Comment:
Why this is mdoified? vload4 will be better for performance?

Reply:
I think @yeonbok 's comment means separate the legacy code and the blocked format code using some jin constants as follows:
```c++
#ifdef USE_BLOCKED_FORMAT
        INPUT_TYPE4 res = vload4(ROI_COUNT * cls + idx, refined_boxes);
        output_boxes[idx0] = res[0];
        output_boxes[idx1] = res[1];
        output_boxes[idx2] = res[2];
        output_boxes[idx3] = res[3];
        output_scores[i] = score;
        output_classes[i] = cls;
#else
        vstore4(vload4(ROI_COUNT * cls + idx, refined_boxes), i, output_boxes);
        output_scores[idx_i4] = score;
        output_classes[idx_i5] = cls;
#endif
```

@yeonbok is it right?

Reply:
yes

Reply:
@OlehKravchyshyn, please separate the legacy code and blocked format code

Reply:
but despite these changes we haven't received performance regression, haven't we?

Reply:
This code changes will affect blocked format as well as other formats.
Did you have performance regression test for other formats except blocked format too?

Reply:
the code didn't support blocked format before, so as I know @wilson-seok  checked all the formats, here is comment from him
https://github.com/openvinotoolkit/openvino/pull/12561#issuecomment-1219479180

Reply:
I just checked in8 model for blocked format. We need to check non blocked format impact on those 7 models which has EDDO op.
![image](https://user-images.githubusercontent.com/1720339/198246038-1c999bc4-bc3b-4371-b2bb-8c82e744322b.png)

Reply:
It is necessary to check regression, but I think it is also necessary to sperate block format code from legacy code for the ㅠbackward compatibility of non-blocked format.

Reply:
Even though the perf regression is not found from the test, the test is not exhaustive, so the original behavior is not expected to be changed if not necessary..

Reply:
This model has performance regression. It's fp16 model. I'll share the model with @mhnap-lohika.
![image](https://user-images.githubusercontent.com/1720339/198470805-b7cd3805-baf5-417b-b127-4c8bffa19650.png)

Reply:
And this model has cldnn program build error. WIll share the model together.
![image](https://user-images.githubusercontent.com/1720339/198474124-9f8c5faa-9a40-486d-835c-07210b133838.png)

Reply:
@OlehKravchyshyn please separate code blocked format code from legacy codes using jit constant.

Reply:
@wilson-seok, separation codes are updated. Could you please run regression test again?

Reply:
No regression in faster_rcnn_resnet50_fpn_coco.
cldnn program build error in instance-segmentation-security-0050 disappeared also.
http://ikswven004.ikor.intel.com:8080/job/i9-12900k-alderlake-05-fp16-32-dlbenchmark-compare-two-branches/54/artifact/regression/regression_public.txt/*view*/